### PR TITLE
CAMEL-21280: azure-files: Only CredentialType.SHARED_ACCOUNT_KEY is handled

### PR DESCRIPTION
**Pull request by s-clauw**
# Description

# Target

- [x] I checked that the commit is targeting the correct branch (note that Camel 3 uses `camel-3.x`, whereas Camel 4 uses the `main` branch)

# Tracking
- [x] If this is a large change, bug fix, or code improvement, I checked there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).

# Apache Camel coding standards and style

- [x] I checked that each commit in the pull request has a meaningful subject line and body.

- [x] I have run `mvn clean install -DskipTests` locally from root folder and I have committed all auto-generated changes.

### CODE REVIEW

File: components/camel-azure/camel-azure-files/src/test/java/org/apache/camel/component/file/azure/FilesOperationsTest.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
+package org.apache.camel.component.file.azure;
```
Comment (oscerd) [inline components/camel-azure/camel-azure-files/src/test/java/org/apache/camel/component/file/azure/FilesOperationsTest.java:1]: Please add the license header

**Review by oscerd — CHANGES_REQUESTED** [review]
Just a minor finding.

**Comment by s-clauw** [comment]
Not sure how the merge to 4.8.x will be done, is it by you or should I also do it there?

**Comment by oscerd** [comment]
We'll backport on 4.8.x

File: components/camel-azure/camel-azure-files/src/test/java/org/apache/camel/component/file/azure/FilesOperationsTest.java (hunk 1 above)
Comment (s-clauw) [reply components/camel-azure/camel-azure-files/src/test/java/org/apache/camel/component/file/azure/FilesOperationsTest.java:17]: Done

**Review by davsclaus — APPROVED** [review]

**Pull request merged**